src/Features/CSharp/Portable/MetadataAsSource/CSharpMetadataAsSourceService.cs

```diff
@@ -68,12 +68,13 @@ protected override async Task<Document> ConvertDocCommentsToRegularCommentsAsync
         }
 
         protected override ImmutableArray<AbstractReducer> GetReducers()
-            => ImmutableArray.Create<AbstractReducer>(
+            => [
                 new CSharpNameReducer(),
                 new CSharpEscapingReducer(),
                 new CSharpParenthesizedExpressionReducer(),
                 new CSharpParenthesizedPatternReducer(),
-                new CSharpDefaultExpressionReducer());
+                new CSharpDefaultExpressionReducer(),
```

Comment:
>new CSharpDefaultExpressionReducer(), [](http://example.com/codeflow?start=16&length=37)

I assume the fixer is adding these commas as part of the formatting?

Reply:
yes.  if it's multi-line, we believe it's better to let all lines have a trailing comma.